modules/exploits/linux/http/freepbx_hotelwakeup_rce.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => 'FreePBX < 13.0.188.1 Remote root exploit',
+                      'Description'     => '
+                        This module exploits an unauthenticated remote command execution in FreePBX module Hotelwakeup
```

Comment:
according to http://wiki.freepbx.org/display/FOP/2016-08-09+CVE+Remote+Command+Execution+with+Privileged+Escalation module versions that are vuln: 13.0.1alpha2 and 13.0.14
and freepbx versions: < 13.0.15, >12